utils/vale/styles/Crossplane/crossplane-words.txt

```diff
@@ -63,4 +63,9 @@ XRC
 XRD
 XRD's
 XRDs
-XRs
\ No newline at end of file
+XRs
+P&T
```

Comment:
```suggestion
```

I think we should write out "patch and transform" instead of the shorthand P&T. If you're not familiar with the shorthand it's hard to figure out what the doc is talking about.